python/lsst/ci/hsc/__init__.py

```diff
@@ -1 +1,6 @@
+try:
+    import SCons.Action
+except ImportError:
+    raise ImportError("lsst.ci.hsc cannot be imported outside of a scons script.")
+
```

Comment:
This seems like quite an arbitrary restriction: although I don't currently have a use case for importing this outside SCons, I'm not sure why we should disallow it. Can you simply move the use of `FunctionAction` into the `SConstruct` file? `makeScons()` is no longer a class method but a free function, you change the way it's called (`RawValidation.makeScons(REPO, data.dataId)` becomes `makeScons(RawValidation, REPO, data.dataId)`, etc), and this error is no longer necessary.